### PR TITLE
Migrate win computer description

### PR DESCRIPTION
**Pull request by amitosw15**
##### SUMMARY

The history from the community repository was preserved. The only change is in the integration test: two task names were duplicated, so they were updated to match the commands.

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME

migrated module 'win computer description' from community to offical

### CODE REVIEW

File: plugins/modules/win_computer_description.py
Comment (Yaish25491) [inline plugins/modules/win_computer_description.py:1]: Add version 2.7 in the description

File: tests/integration/targets/win_computer_description/defaults/main.yml
Comment (Yaish25491) [inline tests/integration/targets/win_computer_description/defaults/main.yml:1]: Please change the parameters to be understandable names. 
example: test_organization: iddqd --> test_organization: organization name 1

Comment (amitosw15) [reply tests/integration/targets/win_computer_description/defaults/main.yml:1]: done https://github.com/ansible-collections/ansible.windows/pull/714/commits/9ae2ead1d5543eda1538664aa3ef31610dad0453

File: plugins/modules/win_computer_description.py
Comment (amitosw15) [reply plugins/modules/win_computer_description.py:1]: https://github.com/ansible-collections/ansible.windows/pull/714/commits/8ea3756f9adb5c046a45201b470e000fdd9053a1

**Review by jborean93 — APPROVED** [review]

**Pull request merged**